<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>920</width>
      <height>460</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>920</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>920</width>
       <height>30</height>
      </rect>
     </property>
     <widget class="QPushButton" name="close">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="min">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="menu">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="theme">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="icon" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>360安全卫士11</string>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="tbHome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbHorse">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbClear">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbFix">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbYouhua">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbActions">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbMarket">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbApp">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="HoverPushButton" name="pbIcon">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>50</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="HoverPushButton" name="pb360">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>50</y>
       <width>90</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>阿斗制</string>
     </property>
    </widget>
    <widget class="HoverPushButton" name="pbLogion">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>80</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>登陆360账号</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="backgroundWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>920</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>920</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <zorder>backgroundWidget</zorder>
   <zorder>widget</zorder>
   <zorder>mainWidget</zorder>
   <zorder>topWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HoverPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">customviews/hoverpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
